### DISCUSSION TITLE
How do you close issues?

### DISCUSSION THREAD

**Discussion by antonmedv** [livewire/livewire#8795] [1. Help]
After you blog (about $1M), I'm thinking to follow your steps with deployer.org. So trying to understand how issues works here.

**Comment by danie-ramdhani** [comment]
@antonmedv  First you hear this:

https://notesonwork.transistor.fm/episodes/whos-responsibility-are-github-issues

Then this:

https://notesonwork.transistor.fm/episodes/still-loving-github-discussions

And this:

https://notesonwork.transistor.fm/episodes/no-you-fix-it

Hope it helps and you can close this.

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147
